src/engine/IndexScan.h

```diff
@@ -167,10 +168,9 @@ class IndexScan final : public Operation {
   // Get the `IdTable` for this `IndexScan` in one piece.
   IdTable materializedIndexScan() const;
 
-  // Get the mapping for all `Variable` values mapped to sorted `ColumnIndex`
-  // values.
-  ad_utility::HashMap<Variable, ColumnIndex> getVariableToSortedIndexMap()
-      const;
+  // Get the first sorted 'Variable' with corresponding `ColumnIndex`.
+  std::optional<std::pair<Variable, ColumnIndex>>
+  getFirstSortedtVariableWithColumnIndex() const;
```

Comment:
This name has a typo, and also check my comment to the previous version of the code for good names and comments.